docs/src/app/components/prop-type-description.css

```diff
@@ -21,18 +21,18 @@
 
 .propTypeDescription table td {
   color: #266d90;
-  font-family: Menlo,Monaco,Consolas,"Courier New",monospace;
-  font-size: 90%;
+  font-family: "Roboto", sans-serif;
```

Comment:
I'm not sure to understand. I think that the `Menlo,Monaco,Consolas,"Courier New",monospace` font family is great for the type (`bool`, `function`, etc). For others, Roboto should be fine.